jstools/merge.py

```diff
@@ -246,11 +246,11 @@ def cat_run(self, outfile, sections, uncompressed=False, strip_deps=True, compre
         license = StringIO()
         seen = []
         for name in newfiles:
-            print >> catted, cat[name]
+            print(cat[name], file=catted)
```

Comment:
FYI this breaks geOrchestra builds (this syntax does not pass under python 2.7.x), related downstream issue: https://github.com/georchestra/georchestra/issues/1773

Reply:
You can add this:
```python
from __future__ import print_function
```

Reply:
Why not using Python 3?